src/modules/emotionHandler/emotionHandler.h

```diff
@@ -0,0 +1,50 @@
+#ifndef __EMOTIONHANDLER__
+#define __EMOTIONHANDLER__
+
+#include <yarp/os/Vocab.h>
+#include <yarp/os/RFModule.h>
+#include <yarp/os/ResourceFinder.h>
+#include <yarp/os/Bottle.h>
+#include <yarp/os/RpcServer.h>
+#include <yarp/os/BufferedPort.h>
+
+using namespace yarp::os;
+using namespace std;

```

Comment:
Never ever use `using namespace in a header file.

Reply:
Fixing 👍